jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/JSON.java

```diff
@@ -6,6 +6,7 @@
 
 import com.fasterxml.jackson.core.*;
 import com.fasterxml.jackson.core.io.SegmentedStringWriter;
+import com.fasterxml.jackson.core.json.PackageVersion;
```

Comment:
I think you added this import by accident - can you remove it?

Reply:
Done!